models/trimmer_decorator.rb

```diff
@@ -0,0 +1,9 @@
+require_relative 'base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    return @nameable.correct_name[0..9] unless super.length <= 10
```

Comment:
- [ ] I get the same error as in the Capitalize class I highlighted due to the use of a non-existent `@nameable`. Kindly rename this variable to `nameable_obj`, or fix the one in the BaseDecorator to match this one. Enter everything works as expected.